trulens_eval/examples/expositional/frameworks/nemoguardrails/iterating_guardrails_apps.ipynb

```diff
@@ -0,0 +1,1250 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #64.      if $groundedness_result &lt; 0.7
</pre><p>Confused here. <code>$groundedness_result, $answerrelevance_result, $contextrelevance_result</code>  have not been defined. Also unclear when the prior two flows get executed. Perhaps the code text is oudated?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
This config specification determines what "flows" get executed when the bot wants to output something:

```yaml
output:
  flows:
    - check language match
    # triad defined seperately so hopefully they can be executed in parallel
    - check rag triad groundedness
    - check rag triad relevance
    - check rag triad qs_relevance
    - bot: explain usage and capabilities
```

The last flow in the notebook is `check rag triad context_relevance` which is now not executed as it is not in the list. Even if it were in the list, it is unclear how the values that were set by the other flows are set or whether they are available in that flow (I'm uncertain if colang will make the variables defined in one flow available in another). The other issue is even if variables were available, you would need to make sure the 2 flows that compute `$groundedness_result` and `$answerrelevance_result`  are executed before they are used in the `check rag triad context_relevance` flow.